tests/template_backends/test_dummy.py

```diff
@@ -55,6 +56,22 @@ def test_html_escaping(self):
         self.assertIn('&lt;script&gt;', content)
         self.assertNotIn('<script>', content)
 
+    def test_django_html_escaping(self):
+        if self.backend_name == 'dummy':
+            self.skipTest("test doesn't apply to dummy backend")
+
+        class TestForm(Form):
+            test_field = CharField()
+
+        media = Media(js=['my-script.js'])
+        form = TestForm()
+        template = self.engine.get_template('template_backends/django_escaping.html')
```

Comment:
I'd say it's besser to keep them as files, because right now they just happen to be identical.
When django someday supports a new Template backend that doesn't use `{{ var }}` syntax it's easier to separate the backends with different directories.